### ISSUE TITLE
How to visualize the layers of DOPE ?

### ISSUE THREAD

**Issue by ArghyaChatterjee** [NVlabs/Deep_Object_Pose#295]
Hi,

I was trying to visualize the layers for beliefmaps and vector field generation part for object keypoint detection. Specially a diagram like this what you did in the CenterPose. Do you have something similar somewhere for DOPE (hand drawn will also work) ?

![Screenshot from 2023-03-07 07-55-34](https://user-images.githubusercontent.com/28845357/230753328-fd3b980e-ae54-4174-9684-4d3e4eee61f4.png)

**Comment by TontonTremblay** [comment]
Yeah there is an option on the inference to have `--show_belief` this will not show the aff though. I had some code to show also the aff. but I am not sure if it is on the repo. The idea is to do something along these lines: https://stackoverflow.com/questions/28898346/visualize-optical-flow-with-color-model should be somewhat easy. I also had code that would show lines. I am not sure if this is in the repo as well. If you are interested in playing with this I can point to you where you could add this code.
